### ISSUE TITLE
Pasting a an odd CR+LF from website crashes Geany (possibly from an odd encoding)

### ISSUE THREAD

**Issue by poikilos** [geany/geany#2668]
- Geany 1.36
- en_US.UTF-8
- GTK 3.24.23
- GLib 2.64.6

Copying and pasting these four characters crashes Geany 1.36 consistently:
![image](https://user-images.githubusercontent.com/7557867/99807263-40ffb080-2b0d-11eb-9a39-944932d5982d.png)

- Opening the file doesn't crash Geany, but _pasting_ it does.

Steps to reproduce issue
1. Copy from Original source: Mailing address at bottom of <https://videocityinc.com/edit/>
   - optional: Paste into a new text file in `nano` to reproduce issue later (copy from there instead of step 1)
     - copy the whole thing from nano or copy just the 4 characters (using GHex)
2. Paste it into any open text file in Geany.

**Comment by b4n** [comment]
I can't reproduce in 1.37/current master.  What you have is [zero-width-space (U+200B)](https://en.wikipedia.org/wiki/Zero-width_space) followed by a newline (U+0A).  Nothing too weird here, and nothing that I would expect Geany to crash on.

@poikilos do you have any plugins enabled? In any case, could you try with a clean configuration to see if you still reproduce (you can use `geany -c /tmp/temp-conf`)?

FWIW, here's the bytes:
```
​
```
(yes, not much to see, but it's there)

**Comment by poikilos** [comment]
> geany -c /tmp/temp-conf

Yep, no crash. On my first try I found that which plugin to disable
- regular mode: prevent crash by disabling: spell check 
  - also I have: Addons, File Browser, Save Actions
  - if I have spell check disabled, then restart, no crash; enable it, paste, instant crash
- using temp-conf: doesn't crash if spell check is enabled (or even if all of the above are enabled too)

My plugins are from Fedora 32:
```
$ dnf list installed | grep geany
geany.x86_64                                      1.36-3.fc32                            @updates                                       
geany-libgeany.x86_64                             1.36-3.fc32                            @updates                                       
geany-plugins-addons.x86_64                       1.36-4.fc32                            @updates                                       
geany-plugins-common.x86_64                       1.36-4.fc32                            @updates                                       
geany-plugins-spellcheck.x86_64                   1.36-4.fc32                            @updates
```

How could it not crash with the temp config even if I enable everything?
